packages/garbo/src/tasks/daily.ts

```diff
@@ -611,6 +614,25 @@ const DailyTasks: GarboTask[] = [
     do: () => use($item`portable steam unit`),
     spendsTurn: false,
   },
+  {
+    name: "Clan Fortune Consults",
+    ready: () =>
+      have($item`Clan VIP Lounge key`) &&
+      getClanLounge()["Clan Carnival Game"] !== undefined &&
+      isOnline("CheeseFax") &&
+      Clan.getWhitelisted().find(
+        (c) => c.name === "Bonus Adventures from Hell",
+      ) !== undefined,
+    completed: () => get("_clanFortuneConsultUses") >= 3,
+    do: (): void => {
+      Clan.with("Bonus Adventures from Hell", () =>
+        cliExecute("fortune 3038166"),
```

Comment:
either do like ```
cliExecute(`fortune ${getPlayerId("cheesefax")}`)
```
or just add a comment saying it's cheesefax.

Also, isn't cheesefax dead?

Reply:
I thought its death was only temporary but who knows.